code/game/objects/items/rogueweapons/melee/swords.dm

```diff
@@ -615,3 +615,31 @@
 	smelt_bar_num = 2
 	max_blade_int = 150
 	max_integrity = 200
+
+/obj/item/rogueweapon/sword/long/blackflamb
+	force = 20
+	force_wielded = 32
+	possible_item_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust, /datum/intent/sword/strike)
+	gripped_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust, /datum/intent/sword/strike, /datum/intent/sword/chop)
+	icon_state = "blackflamb"
+	icon = 'icons/roguetown/weapons/64.dmi'
```

Comment:
this and a couple of other lines are inherited from the parent, you don't need them. this item can be condensed to:
```suggestion
	/obj/item/rogueweapon/sword/long/blackflamb
	force = 20
	force_wielded = 32
	icon_state = "blackflamb"
	name = "blacksteel flamberge"
	desc = "A strange sword with a winding blade forged of blacksteel and a rontz pommel."
	smeltresult = /obj/item/ingot/blacksteel
	max_integrity = 200
```